matRadGUI.m

```diff
@@ -1748,7 +1748,6 @@ function btnuiTableAdd_Callback(hObject, ~, handles)
 data{sEnd+1,2} = 'Select VOI Type';
 data{sEnd+1,3} = 2;
 data{sEnd+1,4} = 'Select obj func/constraint';
-data{sEnd+1,6} = '';
```

Comment:
Agree to that one but that triggers the following question: Should we leave the overlap priority also empty? That would also make the user think what to specify here and not accept a default value of 2 without think through what this means...